src/main/kotlin/com/openlattice/mechanic/upgrades/AddDbCredUsernames.kt

```diff
@@ -43,6 +43,9 @@ class AddDbCredUsernames(
     }
 
     private fun getUsername(userId: String, index: Int): String {
+        if(userId.startsWith("ol-internal|organization|")) {
```

Comment:
`"ol-internal|organization|"` should be a constant.

Reply:
yup.